.github/workflows/ci.yml

```
@@ -26,22 +59,47 @@ jobs:
 
-      - name: 📦 Install dependencies
+      - name: 📦 Install frontend dependencies
         run: yarn install --frozen-lockfile
 
-      - name: 🧪 Run tests
+      - name: 🧪 Run frontend tests with coverage
         run: yarn test --run --coverage
 
-      - name: 📊 Upload coverage reports
+      - name: 📊 Upload frontend coverage reports
         uses: codecov/codecov-action@v3
```

actionlint (1.7.8)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)